README.md

```diff
@@ -1,163 +1,241 @@
-[![Build Status](https://github.com/LibreSplit/LibreSplit/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/LibreSplit/LibreSplit/actions)
+<p align="center">
+<img src="assets/libresplit.svg" width="100" height="100" align="top"/>
+</p>
+<h1 align="center"><a href="https://libresplit.loomeh.is-a.dev">LibreSplit</a></h1>
+
+<p align="center">
+<img src="https://img.shields.io/badge/Code-00599C?style=for-the-badge&logo=c&logoColor=white"</img>
+<img src="https://img.shields.io/badge/AutoSplitters-000081?style=for-the-badge&logo=lua&logoColor=white"</img>
+<img src="https://img.shields.io/github/stars/LibreSplit/LibreSplit?style=for-the-badge&logo=GitHub"</img>
+<img src="https://img.shields.io/static/v1?label=Made%20with&message=GTK%203.0&color=725d9c&style=for-the-badge&logo=GTK&logoColor=white"/>
+<img src="https://img.shields.io/github/license/LibreSplit/LibreSplit?label=license&style=for-the-badge&logo=GNU&logoColor=white&color=b85353"/>
+<p>
+
+<p align="center">
+<a href="https://discord.gg/qbzD7MBjyw"><img src="https://img.shields.io/discord/1381914148585078804?style=for-the-badge&logo=Discord&label=LibreSplit&color=7289da"</img></a>
+<a href="https://github.com/LibreSplit/LibreSplit/actions"><img src="https://img.shields.io/github/actions/workflow/status/LibreSplit/LibreSplit/build.yml?style=for-the-badge&logo=GitHub"</img></a>
+</p>
+
+## About
 
 LibreSplit is a speedrun timer based on [urn](https://github.com/3snowp7im/urn) that adds support for Lua-based auto splitters that are easy to port from ASL.
 
-<img src="https://github.com/wins1ey/LibreSplit/assets/34382191/2adfdae5-9a21-4bdf-a4c4-f1d5962a0b63" width=350>
-<img src="https://github.com/wins1ey/LibreSplit/assets/34382191/4455f57a-3d34-4fa3-9dff-2b342b6c56da" width=350>
+<p align="center">
+<img src="https://github.com/wins1ey/LibreSplit/assets/34382191/2adfdae5-9a21-4bdf-a4c4-f1d5962a0b63" width="350">
+<img src="https://github.com/wins1ey/LibreSplit/assets/34382191/4455f57a-3d34-4fa3-9dff-2b342b6c56da" width="350">
+</p>
 
 ### If you are looking for the public repository of splits, autosplitters and themes. They are located [here](https://github.com/LibreSplit/LibreSplit-resources)
 
 ## Features
 
 - **Split Tracking and Timing:** Accurately track and time your speedruns with ease.
 - **Auto Splitter Support:** Utilize Lua-based auto splitters to automate split timing based on in-game events.
-- **Customizable Themes:** Personalize your timer's appearance by creating and applying custom themes.
+- **Customizable Themes:** Customize your timer's appearance by creating and applying custom themes.
 - **Flexible Configuration:** Configure keybindings and various settings to suit your preferences.
+- **Icon support for splits.**
+- **Always on Top support.**
+- **Auto Splitter Support:** Use Lua to write your own auto splitters or easily port them from LiveSplit's ASL.
```

Comment:
auto splitter support is duplicated in this list, maybe something like ingame timer could be in here instead?